src/components/app/App.jsx

```diff
@@ -46,6 +48,7 @@ class App extends Component {
               <Route exact path="/register" component={RegisterContainer} />
               <Route exact path="/" render={this.boardsRender} />
               <Route exact path="/boards/:boardId" render={this.boardRender} />
+              <Route exact path="*" component={NotFound} />
```

Comment:
Because you don't need a container, use directly the `ErrorDisplayer`, or create another component called `NotFound` in the `commons` folder to abstract the error. In all case, don't create a container in `containers`.

Reply:
done